content/post/kubedb-v2022.12.28/index.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Announcing KubeDB v2022.12.28
+date: "2022-12-28"
+weight: 25
+authors:
+- Mehedi Hasan
+tags:
+- cloud-native
+- dashboard
+- database
+- elasticsearch
+- kafka
+- kibana
+- kubedb
+- kubernetes
+- mariadb
+- memcached
+- mongodb
+- mysql
+- opensearch
+- percona
+- percona-xtradb
+- postgresql
+- redis
+- schema-manager
+---
+
+We are pleased to announce the release of [KubeDB v2022.12.28](https://kubedb.com/docs/v2022.12.28/setup/). This post lists all the major changes done in this release since the last release. 
+The release was mainly focused on updating documentations , writing test and bug fixes and improvements. Aside there was some major features like **kafka,** **ProxySQL: Autoscaler,Monitoring,MariaDB and Percona-XtraDB backend** support. Also, new versions for `ElasticSearch` , `MariaDB`, `PgBouncer`, `Kibana` are available.
```

Comment:
-> Also, new versions for `ElasticSearch` , `MariaDB`, `PgBouncer`, `Kibana` are available.
Add PostgreSQL